Resources/Prototypes/Entities/Structures/Piping/Atmospherics/unary.yml

```diff
@@ -356,6 +356,36 @@
   - type: ApcPowerReceiver
     powerDisabled: false
 
+- type: entity
+  parent: GasThermoMachineFreezer
+  id: GasThermoMachineHellfireFreezer
+  name: hellfire freezer
+  description: An advanced machine that cools gas in connected pipes. Has the side effect of chilling the surrounding area. Cold as Hell!
+  components:
+  - type: Sprite
+    sprite: Structures/Piping/Atmospherics/hellfirethermomachine.rsi
+  - type: GasThermoMachine
+    minTemperature: 30.15
```

Comment:
would make more sense for it to be 23.15K, which is around what T3 parts give you and is the minimum frezon cool temperature